orchagent/portsorch.cpp

```diff
@@ -563,6 +565,10 @@ PortsOrch::PortsOrch(DBConnector *db, DBConnector *stateDb, vector<table_name_wi
 
         m_lagIdAllocator = unique_ptr<LagIdAllocator> (new LagIdAllocator(chassisAppDb));
     }
+
+    auto executor = new ExecutableTimer(m_timer, this, "PORT_STATE_POLL");
+    Orch::addExecutor(executor);
+    m_timer->start();
```

Comment:
Please ensure the timer is started only on the first poll config request and stopped when there are no poll requests. Else the orchagent will get unnecessary notifications every second

Reply:
Sure, will do